elements/rh-icon/demo/sets.html

```diff
@@ -0,0 +1,73 @@
+<div id="rh-icon-demo-output"></div>
+
+<script type="module">
+  import "@rhds/elements/rh-icon/rh-icon.js";
+  import "@rhds/elements/rh-cta/rh-cta.js";
+
+  import { render, html } from 'lit';
+  import { repeat } from 'lit/directives/repeat.js';
+
+  async function renderLink() {
+    render(repeat(['link'], () => {
+      return html`
+        <rh-cta href="https://red-hat-icons.netlify.app/">Red Hat Icons sets</rh-cta>
+      `
+    }), document.getElementById('rh-icon-demo-output'));
+  }
+
+  async function renderIcons(iconMap) {
+    render(repeat(Object.keys(iconMap), (set) => {
+      return html`
+        <div id="icons">
+        <h2 id=${set}>${set}</h2>
+        <ul>
+        ${repeat(iconMap[set], icon => `${set}-${icon}`, icon => html`
+          <li>
+            <rh-icon set="${set}" icon="${icon.replace(/\..*$/, '')}"></rh-icon>
```

Comment:
Right now, all these are hidden from AT. Should add `aria-label="..."` + `role="img"`. to each icon.

Reply:
As I said in the previous comment:

> We are fighting the demo page CSS here. I wanted to keep the code displayed ONLY the code needed for rh-icons however there is a grid style that gets in the way. I think we should split this out into a separate issue.  This has more to do with local demo styles than it does with docs.